pkg/kubelet/server/stats/prometheus_resource_metrics.go

```diff
@@ -24,6 +24,77 @@ import (
 	stats "k8s.io/kubernetes/pkg/kubelet/apis/stats/v1alpha1"
 )
 
+// This file contains a series of deprecated metrics which we emit them by endpoint `/metrics/resource/v1alpha1`.
+// These metrics have been adapted to new endpoint `/metrics/resource` as well as new `Desc`s.
+// In general, we don't need to maintain these deprecated metrics any more.
+// TODO(RainbowMango): Remove this file in release 1.20.0+.
```

Comment:
Just to confirm... we want to remove the entire `prometheus_resource_metrics.go` file in 1.20?

Reply:
100% right. Please see comments above
`These metrics have been adapted to new endpoint `/metrics/resource` as well as new `Desc`s.`.

Reply:
And these metrics have almost a copy at [here](https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/metrics/collectors/resource_metrics.go).